### PR TITLE
fix: Send quiz submission notification to only user's department team manager

### PR DESCRIPTION
**Pull request by shubham-v-simformsolutions**
- fix: Send quiz submission notification to only user's department team manager

### CODE REVIEW

File: mod/quiz/locallib.php [hunk 1]
```diff
@@ -1563,7 +1563,10 @@ function quiz_send_confirmation($recipient, $a) {
  * @return int|false as for {@link message_send()}.
  */
 function quiz_send_notification($recipient, $submitter, $a) {
-    global $PAGE;
+    global $PAGE, $DB;
+
+    $submitterentries = $DB->get_record('user_info_data', array('userid' => $submitter->id, 'fieldid' => '1'));
```
Comment (opensourcedept-simformsolutions) [inline mod/quiz/locallib.php:1568]: fieldID = 1 will be always department? it will be more accurate if we search field by like name

Comment (chintandave-simformsolutions) [inline mod/quiz/locallib.php:1568]: fieldID = 1 will be always department? it will be more accurate if we search field by like name

**Review by chintandave-simformsolutions — CHANGES_REQUESTED** [review]

File: mod/quiz/locallib.php (hunk 1 above)
Comment (shubham-v-simformsolutions) [reply mod/quiz/locallib.php:1568]: Yes, fieldID is always 1 for the department.

**Review by chintandave-simformsolutions — APPROVED** [review]
LGTM

**Pull request merged**